### ISSUE TITLE
Building and Running From Source Code Issues With Java 11 Compatibility

### ISSUE THREAD

**Issue by ron-murhammer** [triplea-game/triplea#10204]
@RoiEXLab @DanVanAtta I'm seeing this error for a number of packages when trying to run the latest code from master in eclipse:
`The package javax.annotation is accessible from more than one module: <unnamed>, java.annotation`

Seems to be something related to dependencies and how they are enforced between Java 8 and 11. If I revert to Java 8 compatibility then they go away but I then have other errors as we are using Java 9+ features like "var". How are you getting around this?

**Comment by DanVanAtta** [comment]
I am not getting around this, I do not see this error. 

I'm running Ubuntu with IDEA & setup instructions are seemingly complete:
https://github.com/triplea-game/triplea/blob/master/docs/development/how-to/ide-setup/intellij-setup.md

**Comment by DanVanAtta** [comment]
@jdimeo , @RaiNova , @asvitkine , @ssoloff  do any of you use eclipse & windows?

**Comment by jdimeo** [comment]
Yes I use Eclipse and Windows! this is a side effect of Project Jigsaw (going to the module system in Java 9) though and isn't platform specific AFAIK. Meaning that the solution is likely going to be in a POM somewhere.

**Comment by DanVanAtta** [comment]
@ron-murhammer were you able to get resolution on this?

It seems like there is a problem with the same dependency present from multiple sources. 
https://community.ptc.com/t5/Windchill/Eclipse-JDK8-to-JDK11-Issue-Package-is-accessible-from-more-than/td-p/686246
In that context the error message makes more sense, though I wonder what the 'unnamed' module is.

@jdimeo if the problem is not platform specific, do you know of any indication why the gradle build is able to complete successfully when run via github actions? Did you previously encounter this problem?

**Comment by ron-murhammer** [comment]
@DanVanAtta Nope. Gave up after a bit and just ran it from the CLI since I was just making a small fix.

**Comment by DanVanAtta** [comment]
this should be largely moot now, JDK11 is not supported on 2.7
